packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -520,6 +520,9 @@ export class InterfaceVpcEndpoint extends VpcEndpoint implements IInterfaceVpcEn
 
   /**
    * The DNS entries for the interface VPC endpoint.
+   * Each entry is a combination of the hosted zone ID and the DNS name. 
+   * The entries are ordered as follows: regional public DNS, zonal public DNS, private DNS, and wildcard DNS. 
+   * This order is not enforced for AWS Marketplace services.
```

Comment:
```suggestion
   * This order is not enforced for AWS Marketplace services.
   *
   * The following is an example. In the first entry, the hosted zone ID is Z1HUB23UULQXV 
   * and the DNS name is vpce-01abc23456de78f9g-12abccd3.ec2.us-east-1.vpce.amazonaws.com.
   *
   * ["Z1HUB23UULQXV:vpce-01abc23456de78f9g-12abccd3.ec2.us-east-1.vpce.amazonaws.com",
   * "Z1HUB23UULQXV:vpce-01abc23456de78f9g-12abccd3-us-east-1a.ec2.us-east-1.vpce.amazonaws.com",
   * "Z1C12344VYDITB0:ec2.us-east-1.amazonaws.com"]
   *
   * If you update the PrivateDnsEnabled or SubnetIds properties, the DNS entries in the list will change.
```

Reply:
Don't forget to backquote these code samples for Markdown

Reply:
@rix0rrr can you please clarify what you mean? You mean I should surround the sample with ```...```?